src/main/java/com/labsynch/labseer/dto/BulkLoadParentSaltFormLotDTO.java

```diff
@@ -0,0 +1,108 @@
+package com.labsynch.labseer.dto;
+
+import java.util.Collection;
+import java.util.List;
+import org.apache.commons.lang3.builder.ReflectionToStringBuilder;
+import org.apache.commons.lang3.builder.ToStringStyle;
+import flexjson.JSONDeserializer;
+import flexjson.JSONSerializer;
+
+public class BulkLoadParentSaltFormLotDTO {
```

Comment:
This "Data Transfer Object" (DTO) class is just a data container that we'll use to collect the results of a SQL (really JPQL) query. These are also known as "POJO"s (Plain Old Java Objects).